src/ray/gcs/gcs_server/gcs_job_manager.h

```diff
@@ -24,6 +24,15 @@
 namespace ray {
 namespace gcs {
 
+// Please keep this in sync with the definition in ray_constants.py.
+const std::string kRayInternalNamespacePrefix = "_ray_internal_";
```

Comment:
I think if this file is included twice, it's going to cost duplicate defining of the variables.

These are global variables here.

Reply:
Why not put them into cpp directly if it's not used in other places.

Reply:
Sure, sounds good

Reply:
Actually, it's used in a test file, so I think I need to leave it in this `.h` file (I tried moving it to the `.cc` file and the test wouldn't compile).  Is that okay, or is there another workaround?